samples/saga/ravendb-sequence-generator/Version_5/Sample/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NServiceBus;
+using NServiceBus.Persistence;
+using Raven.Client;
+using Raven.Client.Document;
+using System;
+using System.Threading.Tasks;
+
+class Program
+{
+
+    static void Main()
+    {
+        BusConfiguration busConfig = new BusConfiguration();
+		busConfig.EndpointName( "Samples.RavendbSequenceGenerator" );
+        busConfig.UseSerialization<JsonSerializer>();
+        busConfig.EnableInstallers();
+
+        #region RavenDBSetup
+
+        IDocumentStore defaultStore = new DocumentStore()
+                                      {
+                                          Url = "http://localhost:32076",
+										  DefaultDatabase = "Sample"
+                                      }
+            .Initialize();
+
+        busConfig.UsePersistence<RavenDBPersistence>()
```

Comment:
Not sure about default. Could be good to comment on it when you add `.md` content.